src/integrations/grid-eye/grid-eye.service.ts

```diff
@@ -105,7 +106,33 @@ export class GridEyeService
       temperatures.push(await this.getPixelTemperature(i));
     }
 
-    return math.reshape(temperatures, [8, 8]) as number[][];
+    let grid = math.reshape(temperatures, [8, 8]) as number[][];
+    if (this.config.maskZeroBasedValues) {
+      grid = await this.maskZeroBasedValues(grid);
+    }
+    return grid;
+  }
+
+  /**
+   * Replace 0 based values (0 t0 1) with nearest preceding valid value.
```

Comment:
```suggestion
   * Replace 0 based values (0 to 1) with nearest preceding valid value.
```
typo, also see the comment below